portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['I am left handed', 'My favorite food is raspberries!', 'I speak Chinese and am learning Spanish', 'I can solve a Rubik\'s Cube'];
```

Comment:
=o  I've been learning Chinese and speak Spanish (although not exceptionally... i just had like five years of schooling on it).